src/chordDetail.js

```diff
@@ -116,7 +120,7 @@ class ChordDetail extends Component {
                         >
                             {this.state.classes.map ((cl) => {
                                 return (
-                                    <option key={cl.class} value={cl.id}>{cl.class}</option>
+                                    <option value={cl.class}>{cl.class}</option>
```

Comment:
Remember that any time you are mapping over an array in React JSX, you'll need to give the returned element a unique key so that each rendered element has a unique identifier from the others.